README.md

```diff
@@ -129,3 +129,6 @@ flutter pub run grinder update-code-segments
     * Update the description to include what changes have been done since the
     last release.
     * Publish the release.
+
```

Comment:
Can you replace the instructions in step 4?

Reply:
Done